src/anim/controller/anim-controller.js

```diff
@@ -187,7 +187,7 @@ class AnimController {
 
         const activeClip = this._animEvaluator.findClip(this.activeStateAnimations[0].name);
         if (activeClip) {
-            return time / activeClip.track.duration;
+            return time / activeClip.track.duration * this.activeState.speed;
```

Comment:
Potential divide by 0 error. 

Also, what happens when (if) speed is negative?

Reply:
Shouldn't the duration be longer the slower the speed is? ie, should this be:

```
return time / (activeClip.track.duration / this.activeState.speed);
```